notebooks/03_Distributed_Training/03_02_Distributed_Training_MPI_Horovod.ipynb

```diff
@@ -90,6 +90,32 @@
     "In total, there're 8 gpus form a ring and doing distributed training"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Prerequiste: \n",
+    "1. Update default-editor roles\n",
+    "\n",
+    "    There's an upstream isuse that default-editor doesn't have permission to create mpijobs.\n",
+    "\n",
+    "    Adding this on Clusterrole to skip error: User \"system:serviceaccount:kubeflow:default-editor\" cannot create resource \"mpijobs\" in API group \"kubeflow.org\" in the namespace \"kubeflow\"\n",
+    "\n",
+    "    ```shell\n",
+    "    kubectl edit clusterrole kubeflow-kubernetes-edit -n kubeflow\n",
```

Comment:
You are editing the wrong file.

```
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  creationTimestamp: "2020-03-04T21:03:04Z"
  name: default-editor
  namespace: kf12
  ownerReferences:
  - apiVersion: kubeflow.org/v1
    blockOwnerDeletion: true
    controller: true
    kind: Profile
    name: kf12
    uid: 8a7c8372-5e5b-11ea-8085-021721b51cb6
  resourceVersion: "8128122"
  selfLink: /apis/rbac.authorization.k8s.io/v1/namespaces/kf12/rolebindings/default-editor
  uid: 8a81a2cf-5e5b-11ea-9d96-0a24c0f344ae
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: kubeflow-edit
subjects:
- kind: ServiceAccount
  name: default-editor
  namespace: kf12
```

The `default-editor` SA bindings to ClusterRole `kubeflow-edit`. You need to change that role. Where do you get `kubeflow-kubernetes-edit`?

Reply:
Yes, I should be editing `kubeflow-edit` clusterrole. The reason that `kubeflow-kubernetes-edit` also works because it has the same matchLabel as `kubeflow-edit`: 

```
{"rbac.authorization.kubeflow.org/aggregate-to-kubeflow-edit":"true"}
```

Thus, changing one of them both works. I get `kubeflow-kubernetes-edit` from [here](https://github.com/kubeflow/manifests/blob/master/kubeflow-roles/base/cluster-roles.yaml#L78).

I fix it up in the latest PR.